src/rewrite_clj/node.cljc

```diff
@@ -670,7 +670,8 @@
        n/string)
   ;; => \"\\\"line1\\n\\nline3\\\"\"
   ```"
-  [lines] (rewrite-clj.node.stringz/string-node lines))
+  ([lines] (rewrite-clj.node.stringz/string-node lines))
+  ([lines raw-string] (rewrite-clj.node.stringz/string-node lines raw-string)))
```

Comment:
I don't think the public API needs to know about this second argument.

Reply:
Good point, I don't think my import-vars generation is smart enough to ignore an arity yet. Will look into it.